.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please remove the `.vs/` folder from this PR 

`git rm -r .vs --cached` or something similar would work